src/app/containers/Ad/utilities/adSlotStyles.js

```diff
@@ -0,0 +1,74 @@
+import { css } from 'styled-components';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+// Ad Slot Widths for custom breakpoints
+const MPU_WIDTH_MIN = '18.75rem'; // 300px
+
+const LEADERBOARD_WIDTH_MIN = '20rem'; // 320px
+
+const LARGE_LEADERBOARD_WIDTH_MIN = '58.75rem'; // 940px
+
+// Ad Slot Heights
+/*
+  The height of the Ad Slot Label is taken into account for the total max height
+  of the Ad Slot.
+*/
+const ADSLOT_LABEL_HEIGHT = 1.5625; // 25px
+
+const ADSLOT_HEIGHTS = {
+  GROUP_1: `${3.125 + ADSLOT_LABEL_HEIGHT}rem`,
+  // 50px + ADSLOT_LABEL_HEIGHT = 75px
+  GROUP_2_LEADERBOARD: `${3.75 + ADSLOT_LABEL_HEIGHT}rem`,
+  // 60px + ADSLOT_LABEL_HEIGHT = 85px
+  GROUP_3_LEADERBOARD: `${5.625 + ADSLOT_LABEL_HEIGHT}rem`,
+  // 90px + ADSLOT_LABEL_HEIGHT = 115px
+  GROUP_4: `${15.625 + ADSLOT_LABEL_HEIGHT}rem`,
+  // 250px + ADSLOT_LABEL_HEIGHT = 275px
+};
+
```

Comment:
Although the `GROUP1` and `GROUP2` heights are the same for both ad types, I think it will be better to separate this object into two, one with leaderboard heights and another with the MPU ones in case the change in the future.